src/Compiling/README.md

```diff
@@ -1,26 +1,17 @@
-# Microsoft Azure ApiManagement Policy Toolkit policy compiler tool
+# Azure API Management Policy Toolkit Compiling tool
 
-Microsoft Azure API Management is a hybrid, multicloud management platform for APIs across all environments. As a
-platform-as-a-service, API Management supports the complete API lifecycle.
-
-This library contains .net tool which transforms policies wrote in C# to format accepted by Microsoft Azure Api
-Management.
-
-## Getting started
-
-### Install compiler CLI tool
+This project builds a dotnet tool which can compile C# policy documents into Azure API Management XML (rawxml / Razor)
+policy documents.
 
+## Install
 Install the Microsoft Azure Api Management Policy Toolkit compiler CLI tool with [NuGet][nuget]:
 
-```dotnetcli
+```shell
 dotnet tool install Azure.ApiManagement.PolicyToolkit.Compiling
 ```
 
-### Compile the policy
-
-```bash
-dotnet azure-apim-policy-compiler --s .\PATH\TO\SOURCE\FOLDER --o .\PATH\TO\OUTPUT\FOLDER
-```
+If the package is not available on the public NuGet feed for your environment, you can install it from the repository
```

Comment:
Same here, let's point to where we track it to avoid duplicates and give them place to check status